mobilizon_reshare/storage/db.py

```diff
@@ -65,8 +65,12 @@ async def _implement_db_changes(self):
             logging.info("Updated database to latest version")
 
     async def setup(self):
+        tortoise_config = get_tortoise_orm()
+        Tortoise.init_models(
```

Comment:
Why didn't we need this until now?

Reply:
I think it's connected to initializing the relations for the test client. It needs to happen in a different way